README.md

```diff
@@ -1 +1,90 @@
-# functional-harness
+## testcafe-functional-harness
+
+Runs functional tests in local browsers and on SauceLabs.
+
+#### What it does
+
+1. Starts the server with the test pages.
+2. Opens the specified browsers (locally or on SauceLabs).
+3. Runs mocha tests.
+4. Closes the browsers and stops the server after all tests are finished.
+
```

Comment:
And, I guess, the description lacks something important in the introduction.
What is the thing that we achieve by making such a trick - running testcafe tests from within mocha tests - and that we cannot achieve without this harness? (or is it just me for whom this is unclear?)